themes/default/content/learn/pulumi-fundamentals/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Pulumi Fundamentals"
+layout: module
+date: 2021-09-10T12:00:00-05:00
+draft: true
+description: Use Pulumi to build, configure, and deploy a modern application.
+meta_desc: Use Pulumi to build, configure, and deploy a modern application.
+index: 0
+meta_image: meta.png
+youll_learn:
+    - creating projects
+    - configuring and provisioning infrastructure
+    - exporting outputs
+tags:
+    - fundamentals
+providers:
+    - docker
+block_external_search_index: true
+---
+
+This tutorial module demonstrates how to use Pulumi to build, configure, and
```

Comment:
we might want to drop module from the content pieces entirely and only refer to these as tutortials

Reply:
Yep! I need to copy Aaron's feedback here as he said the same thing. I hadn't realized the names weren't set. Thanks!